### DISCUSSION TITLE
congratulations on your work with apollo and artemis

### DISCUSSION THREAD

**Discussion by dsanttos12** [ClassicOldSong/Apollo#73] [General]
Congratulations on your work. Artemis is better than Moonlight, better than Parsec, better than Steam Link (which was developed by a giant corporation), especially because it has lower latency and excellent stability. I admire your ability to develop software on your own without bugs and very stable, things that large corporations have difficulty doing.

On Artemis and Apollo, I am able to use the native resolution of my Galaxy S20 FE with 120 fps and HDR, through the custom resolution that I created in Artemis. I couldn't use custom resolutions with HDR on NVIDIA Game Stream
The Artemis virtual gamepad is much better than the Moonlight and Steam Link virtual gamepads on Android. When I complained on the Moonlight Discord channel about the virtual gamepad being terrible on Android, the developers said it was pinging and suspended me for 24 hours.
Artemis supports up to 300mbps bitrate, which is great for those with good internet. You can even have custom bitrates, a feature I haven't tested yet, since 300mbps bitrate is excellent for my phone's resolution, while Moonlight only goes up to 150mbps bitrate and Parsec only goes up to 50mbps.
In Artemis, I can use Windows shortcuts directly and use a virtual keyboard, and I can choose between trackpad and direct touch on the mouse.
Apollo and Artemis are excellent. Now answer me: how did you manage to develop stable and complete software like Apollo and Artemis all by yourself. Congratulations, your article must be widely publicized.

**Comment by ClassicOldSong** [comment]
No tricks, just skill issue.
